net.blumia.pineapple.steam-recording-exporter.yml

```diff
@@ -0,0 +1,89 @@
+command: PineappleSteamRecordingExporter
+rename-icon: net.blumia.pineapple.steam-recording-exporter
+
+finish-args:
+  # X11 and Wayland support
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  
+  # Audio/Video support
+  - --socket=pulseaudio
+  - --device=dri
+  
+  # File system access
+  - --filesystem=home:ro
+  - --filesystem=~/.local/share/Steam:ro
+  - --filesystem=~/.var/app/com.valvesoftware.Steam/.local/share/Steam
+  - --filesystem=~/.steam/steam:ro
+  - --filesystem=~/Videos:rw
+  - --filesystem=~/Movies:rw
+  
+  # Network access
+  - --share=network
+  
+  # D-Bus access for desktop integration
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.gtk.vfs.*
+
+modules:
+  #- name: ffmpeg
+  #  buildsystem: autotools
+  #  config-opts:
+  #    - --enable-shared
+  #    - --disable-static
+  #    - --enable-gnutls
+  #    - --disable-doc
+  #    - --disable-programs
+  #    - --disable-encoders
+  #    - --disable-muxers
+  #    - --enable-encoder=libx264
+  #    - --enable-encoder=aac
+  #    - --enable-muxer=mp4
+  #    - --enable-muxer=matroska
+  #    - --enable-protocol=file
+  #    - --enable-protocol=pipe
+  #  sources:
+  #    - type: archive
+  #      url: https://ffmpeg.org/releases/ffmpeg-7.1.1.tar.xz
+  #      sha256: 8684f4b00f94b85461884c3719382f1261f0d9eb3d59640a1f4ac0873616f968
+  #  cleanup:
+  #    - /include
+  #    - /lib/pkgconfig
+  #    - /share/ffmpeg/examples
+
+  - name: qthttpserver
+    buildsystem: cmake-ninja
+    builddir: true
+    post-install:
+      - ln -s ${FLATPAK_ARCH}-linux-gnu/libQt6HttpServer.so  /app/lib/
+      - ln -s ${FLATPAK_ARCH}-linux-gnu/libQt6HttpServer.so.6  /app/lib/
+      - ln -s ${FLATPAK_ARCH}-linux-gnu/libQt6HttpServer.so.6.8.3  /app/lib/
+    sources:
+      - type: archive
+        url: https://invent.kde.org/qt/qt/qthttpserver/-/archive/v6.8.3/qthttpserver-v6.8.3.zip
+        sha256: 6511710dd6b81708d2039628899b67bcafe0eaf87b56558396623d522f7896c9
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - name: PineappleSteamRecordingExporter
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: git
+        url: https://github.com/BLumia/pineapple-steam-recording-exporter.git
+        # branch: master
+        tag: 0.1.0
+        commit: 1f376ca239e8d0cef6576495afae9059fa00a3fc
+    post-install:
+      - install -Dm644 qml/app_icon.png /app/share/icons/hicolor/256x256/apps/net.blumia.pineapple.steam-recording-exporter.png
+      - install -Dm644 qml/app_icon.png /app/share/icons/hicolor/128x128/apps/net.blumia.pineapple.steam-recording-exporter.png
+      - install -Dm644 qml/app_icon.png /app/share/icons/hicolor/64x64/apps/net.blumia.pineapple.steam-recording-exporter.png
+      - install -Dm644 qml/app_icon.png /app/share/icons/hicolor/48x48/apps/net.blumia.pineapple.steam-recording-exporter.png
```

Comment:
this is an Heisenicon. It's all four size at once.


Don't do that.

Reply:
minimum size is 128x128 so this won't work.

Reply:
Then your app is not ready for submission if you can't even provide a 128x128 icon.

Reply:
> also they should be installed by the build system. That's what the build system is for.

Build system actually already did this. PR updated to remove this in manifest file.